src/utils/hwManager.js

```diff
@@ -32,20 +36,70 @@ const getAccountsFromDevice = async ({ device: { deviceId }, networkConfig }) =>
  * signSendTransaction - Function.
  * This function is used for sign a send transaction.
  */
-const signSendTransaction = () => {
-  // TODO implement logic for this function
-  signTransaction();
-  throw new Error('not umplemented');
+const signSendTransaction = async (account, data) => {
+  const transactionObject = createSendTX(
+    account.info.LSK.publicKey,
+    data.recipientId,
+    data.amount,
+    data.data,
+  );
+
+  const transaction = {
+    deviceId: account.hwInfo.deviceId,
+    index: account.hwInfo.derivationIndex,
+    tx: transactionObject,
+  };
+
+  try {
+    const signature = await signTransaction(transaction);
+    const signedTransaction = { ...transactionObject, signature };
+    const result = { ...signedTransaction, id: utils.getTransactionId(signedTransaction) };
+    return result;
+  } catch (error) {
+    return new Error(error);
```

Comment:
```suggestion
    throw new Error(error);
```
If you want to catch, you need to throw it, but I was suggesting not to catch the error at all in https://github.com/LiskHQ/lisk-hub/pull/2453#discussion_r325137875